doc/plugins/request-validation.md

```diff
@@ -67,7 +67,7 @@ curl http://127.0.0.1:9080/apisix/admin/routes/5 -H 'X-API-KEY: edd1c9f034335f13
         "nodes": {
             "127.0.0.1:8080": 1
         }
-    }
+    },
```

Comment:
Changed the wrong place?

Reply:
@spacewander Apologies. Corrected now :) . Please review.